nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/src/main/java/org/apache/nifi/services/azure/storage/AzureStorageConflictResolutionStrategy.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.services.azure.storage;
+
+import com.azure.storage.blob.models.BlobErrorCode;
+import org.apache.nifi.components.AllowableValue;
+
+public enum AzureStorageConflictResolutionStrategy {
+    FAIL_RESOLUTION("fail", "Fail if the blob already exists"),
+    IGNORE_RESOLUTION("ignore",
+            String.format(
+                "Ignore if the blob already exists; the 'azure.error' attribute will be set to the value '%s'",
+                BlobErrorCode.BLOB_ALREADY_EXISTS.toString()
+            )
+    ),
+    REPLACE_RESOLUTION("replace", "Replace blob contents if the blob already exist");
+
+    private final String label;
+    private final String description;
+
+    AzureStorageConflictResolutionStrategy(String label, String description) {
+        this.label = label;
+        this.description = description;
+    }
+
+    public AllowableValue getAllowableValue() {
+        return new AllowableValue(name(), label, description);
+    }
```

Comment:
Instead of adding this method, the `enum` could implement the `DescribedValue` interface, and then the enum can be referenced directly in the PropertyDescriptor Builder.

Reply:
Fixed in bbc984de2e05c902893f42828b7562278a9e2357.